src/Components/ActionsBuilder/SupportedActions/UpdateENSContent/utils.ts

```diff
@@ -0,0 +1,48 @@
+import namehash from 'eth-ens-namehash';
+import contenthash from 'content-hash';
+import {
+  MAINNET_ID,
+  LOCALHOST_ID,
+  GOERLI_ID,
+  ARBITRUM_ID,
+  ARBITRUM_TESTNET_ID,
+} from 'utils';
+
+export const convertToNameHash = (name: string) => namehash.hash(name);
+
+export const convertToContentHash = (ipfsHash: string) =>
+  contenthash.fromIpfs(ipfsHash);
+
+export const convertToIpfsHash = (contentHash: string) => {
+  if (!contentHash) return null;
+  return contenthash.decode(contentHash);
+};
+
+export const getIpfsUrl = (ipfsHash: string) => {
+  const ipfsRoot = 'ipfs://';
+  return `${ipfsRoot}${ipfsHash}`;
+};
+
+export const isValidChainId = (chainId: number) => {
+  if (!chainId || chainId === LOCALHOST_ID) {
+    return MAINNET_ID;
+  } else {
+    return chainId;
+  }
+};
+
+export const getBlockChainUrl = (chainId: number, address: string) => {

```

Comment:
We now have a utility function to do this in `src/provider/chains.ts`. (`getBlockExplorerUrl`) It uses wagmi's internal config to generate the block explorer URL. Can we use that instead?

Reply:
Yes, much better!